app/views/judgment/judgmentHelp.scala.html

```diff
@@ -0,0 +1,397 @@
+        <h1 class="govuk-heading-l">Transferring Judgments to The National Archives</h1>
+
+        <h2 class="govuk-heading-m">A step-by-step guide to using Transfer Digital Records (TDR)</h2>
+        <nav aria-label="List">
+          <ul class="govuk-list govuk-list--bullet">
+            <li><a href="#about">About this service</a></li>
+            <li><a href="#before-you-begin">Before you begin</a></li>
+            <li>
+              <a href="#register-new-account">Registering for a new account</a>
+              <ul class="govuk-list govuk-list--bullet">
+                <li><a href="#new-user-email">New user email</a></li>
+                <li><a href="#two-step-auth">Two-step authentication</a></li>
+                <li><a href="#set-pass">Set your password</a></li>
+              </ul>
+            </li>
+            <li>
+              <a href="#existing-account-holder">Using this service as an existing account holder</a>
+              <ul class="govuk-list govuk-list--bullet">
+                <li><a href="#accessing-tdr">Accessing TDR</a></li>
+                <li><a href="#step-0">Step 0: Sign in to your account using two-step authentication</a></li>
+                <li><a href="#step-1">Step 1: Check your file before uploading</a></li>
+                <li><a href="#step-2">Step 2: Upload the judgment</a></li>
+                <li><a href="#step-3">Step 3: Upload undergoes checks</a></li>
+                <li><a href="#step-4">Step 4: Transfer complete</a></li>
+              </ul>
+            </li>
+            <li><a href="#glossary">Glossary</a></li>
+          </ul>
+        </nav>
+        <br>
+
+        <h2 class="govuk-heading-m" id="about">About this service</h2>
+        <p class="govuk-body">
+          Transfer Digital Records (TDR) is The National Archives’ secure digital transfer service that enables Public Record Bodies to upload, prepare and then export selected digital public records in order to transfer them to The National Archives (TNA) for permanent archiving and publication.
+        </p>
+        <p class="govuk-body">
+          For anyone involved in recording judgments, this service is used to send the final approved judgment to TNA for preservation and publication. This is a two stage process:
+        </p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>First you will upload your judgment into TDR </li>
+          <li>Then you will be prompted to export your judgment for transfer to TNA</li>
+        </ul>
+
+        <h2 class="govuk-heading-m" id="before-you-begin">Before you begin</h2>
+
+        <h3 class="govuk-heading-s">Web browsers</h3>
+        <p class="govuk-body">
+          We do not support Internet Explorer (IE11). We advise you to use a different browser such as Edge, Chrome, Safari or Firefox to ensure you can use this service and all of its features.
+        </p>
+
+        <h3 class="govuk-heading-s">Secured access</h3>
```

Comment:
Should we add ids to the h3 headings as well, in case we want to link them?

Reply:
:smile: Yeah. I've added an id to all header elements that didn't have one.